clang-tools-extra/test/clang-tidy/checkers/cppcoreguidelines/pro-type-const-cast.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %check_clang_tidy -check-suffix=STRICT  %s cppcoreguidelines-pro-type-const-cast %t -- -config="{CheckOptions: {StrictMode: true}}"
+// RUN: %check_clang_tidy -check-suffix=STRICT  %s cppcoreguidelines-pro-type-const-cast %t -- -config="{CheckOptions: {cppcoreguidelines-pro-type-const-cast.StrictMode: true}}"
```

Comment:
This is interesting: the check option was actually something in between global and local, since it's inside the `CheckOptions` block, instead of at the very top-level of the .clang-tidy file.

Reply:
IMO, we should define check related config here instead of very top level. what i mention global options is also in this scopes instead of clangtidy's command line option.